server/lib/sentry/index.ts

```diff
@@ -180,7 +180,7 @@ export const reportErrorToSentry = (err: Error, params: CaptureErrorParams = {})
 
     if (checkIfSentryConfigured()) {
       Sentry.captureException(err);
-    } else {
+    } else if (config.sentry?.logToConsole) {
```

Comment:
Are thrown exceptions (e.g. in graphql resolvers) still being logged in console? It's important to have them

Reply:
They should, this is the function that `reportErrorToSentry` not that logs it to the console...

Reply:
@kewitz Some places in the code are not logging to the console, they're just throwing the error at Sentry. And `logger` doesn't always show the stack trace. That's why this helper was introduced. I'm a bit wary that this change could invisibilize some errors in dev, but no blocker at that point since most of them have a `logger.error` next to them.

Ideally, I would love to see an API like:

```ts
reportErrorToSentry = (error, { log = true } = {}) => {}
```

Then instead of:

```ts
catch (e) {
  logger.error(`Something happened`, e);
  reportErrorToSentry(e);
}
```

We would simply write:

```ts
catch (e) {
  reportErrorToSentry(e);
}
```

Or, if we don't want to log anything:

```ts
catch (e) {
  reportErrorToSentry(e, { log: false });
}
```

Reply:
https://github.com/opencollective/opencollective-api/pull/10638